src/updater-builder.ts

```diff
@@ -42,6 +42,7 @@ export class UpdaterBuilder {
         `DEPENDABOT_JOB_ID=${this.jobParams.jobId}`,
         `DEPENDABOT_JOB_TOKEN=${this.jobParams.jobToken}`,
         `DEPENDABOT_JOB_PATH=${JOB_INPUT_PATH}/${JOB_INPUT_FILENAME}`,
+        `DEPENDABOT_OPEN_TIMEOUT_IN_SECONDS=10`,
```

Comment:
The timeout is 15s in the connectivity check so it might be better to match that here?

Reply:
Thanks for catching that! I should have looked at your PR again 🤦